src/lib.rs

```diff
@@ -61,35 +77,27 @@ fn translate_level(level: usize) -> Option<ColorLevel> {
 
 fn supports_color(stream: Stream) -> usize {
     let force_color = env_force_color();
-    let no_color = match std::env::var("NO_COLOR") {
-        Ok(val) if val == *"0" => false,
-        Ok(_) => true,
-        Err(_) => false,
-    };
-    let min = std::cmp::max(force_color, 0);
```

Comment:
wait, I have a question. I think I'm missing something. Why is this guaranteed to be 0 anywhere where it's used?

Reply:
ah nevermind. Literally the first `if` clause lol.